TradingBotTests/MarketDataSourceTests.cs

```diff
@@ -72,18 +74,21 @@ public async Task ShouldCorrectlyReturnPrices()
                 Close = 13m,
                 High = 14m,
                 Low = 11m,
-                Volume = 100m
+                Volume = 100m,
+                TradedValue = 0m,
+                FearGreedIndex = 50m
             }
         });
 
-        _marketDataCache.Received(1).CacheValidSymbols(Arg.Is<IReadOnlyList<TradingSymbol>>(symbols =>
+        _marketDataCache.Received(2).CacheValidSymbols(Arg.Is<IReadOnlyList<TradingSymbol>>(symbols =>
```

Comment:
This should probably be `1`. There's no need to get the same data from Alpaca twice. Same with bar requests in `ShouldCorrectlyInitializeCacheForBacktest()`.